app/javascript/shared/transfersLayouts.js

```diff
@@ -130,6 +155,45 @@ const buildSequentialTransfersArray = function(transferRequests) {
   return transfers
 }
 
+
+// Gets a list of requests and create as many destinations as different library types we have defined in the requests.
+// For every source plate a number of wells at the destination is assigned so the wells will be transferred as stamp into 
+// those list of wells reserved in column order. Eg:
+// Default value is 24 wells, so first plate will get the first 3 columns, second plate columns 4 to 6, etc; the position
+// will be a stamp from the source into those columns (so A2 in plate 2 will go to A5 in each destination plate)
+/* Args: 
+ * - transferRequests: each request from a source plate that we want to connect with a destination plate
+ * - numberOfWellsForEachSourcePlateInColumnOrder: number of wells that every source plate will have reserved in each
+ * destination plate (default 24 wells (3 columns))
+*/
+const buildSequentialLibrarySplitTransfersArray = function(transferRequests, numberOfWellsForEachSourcePlateInColumnOrder=24) {
+  const transfers = new Array(transferRequests.length)
+  const libraryTypes = []
+  const positionWellInLibraryTypePlate = {}
+  for (let i = 0; i < transferRequests.length; i++) {
```

Comment:
Feels like a map would be better here. Gets a little complicated with `positionWellInLibraryTypePlate` but as best I can tell that isn't actually used for anything?